earth_enterprise/rpms/buildSrc/src/main/groovy/com/tsciences/os/Platform.groovy

```diff
@@ -0,0 +1,64 @@
+package com.tsciences.os
+
+class Platform {
+    static private String osId = null;
+    static private String osVersionId = null;
+
+    static def obtanOsAndVersionIds() {
+        osId = null
+        osVersionId = null
+
+        if (new File('/etc/os-release').exists()) {
```

Comment:
os-release actually is a good choice to use.  While not yet a concern, I did test and it does produce a valid result on ubuntu.